### ISSUE TITLE
Range.detach is now a no-op

### ISSUE THREAD

**Issue by thomasbachem** [timdown/rangy#223]
Chrome 36 shows the following warning in the console when using rangy:

> 'Range.detach' is now a no-op, as per DOM (http://dom.spec.whatwg.org/#dom-range-detach).

Triggered by:

``` javascript
if (isHostMethod(document, "createRange")) {
    testRange = document.createRange();
    if (areHostMethods(testRange, domRangeMethods) && areHostProperties(testRange, domRangeProperties)) {
        implementsDomRange = true;
    }
    testRange.detach(); // <-- TRIGGER
}
```
